server/database.js

```diff
@@ -73,3 +100,32 @@ export const deleteReservation = async (id) => {
   db.chain.get("reservations").remove({ id }).value();
   await db.write();
 };
+
+export const getConfigFields = async () => {
+  db.chain = lodash.chain(db.data);
+  return db.chain.get("config");
+};
+
+export const addConfigFields = async (fields) => {
+  db.chain.get("config").get("fields").chain().assign(fields).value();
+  await db.write();
+};
+
+export const getAdditionalFields = async (id, reservationUpdated) => {
+  db.chain = lodash.chain(db.data);
+  const configFields = db.chain.get("config").get("fields");
+  const filterFields = configFields.chain().filter(field => {
```

Comment:
You could replace this to `configFields.chain().filter(field=> field.additional).value()`